src/style.css

```diff
@@ -0,0 +1,86 @@
+body {
+  margin: 0;
+}
+
+.flexbox {
+  display: flex;
+  flex-wrap: wrap;
+  --box-width-l: calc(100vw / 3);
+  --box-width-m: calc(100vw / 2);
+  --box-width-s: calc(100vw);
+}
+
+.box {
+  min-width: 300px;
+
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  height: 300px;
+  text-align: center;
+  line-height: 300px;
+}
+
+.box--1 {
+  background-color: rgb(100%,0%,0%);
+  order: 1;
+}
+
+.box--2 {
+  background-color: rgb(80%,0%,0%);
+  order: 2;
+}
+
+.box--3 {
+  background-color: rgb(60%,0%,0%);
+  order: 3;
+}
+
+.box--4 {
+  background-color: rgb(40%,0%,0%);
```

Comment:
It is not necessary to use rgb here, becouse in this case not fourth parameter for rgb function.
Better hah color.